leapp/cli/upgrade/__init__.py

```diff
@@ -91,5 +91,14 @@ def upgrade(args):
             msg = 'No such Actor: {}'.format(actor_name)
             logger.error(msg)
             raise CommandError(msg)
-    workflow.run(context=context, skip_phases_until=skip_phases_until)
+    try:
+        workflow.run(context=context, skip_phases_until=skip_phases_until)
+    except LeappRuntimeError as e:
+        msg = '{} - Please check the exception above'.format(e.message)
```

Comment:
I think the exception handler should be kept in one place (`leapp.utils`) and imported where needed.

Reply:
Good idea, and that could be even done with some kind of connection manager
```python
with exception_beautifier():
    workflow.run(context=context, skip_phases_until=skip_phases_until)
```

Reply:
Ack, thanks for the ideas...